modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -51,19 +48,180 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         authService.handleAuthResult(data);
     }
 }
+----
++
+NOTE: This handler should always invoke `handleAuthResult`, providing the `Intent`. This will exchange the temporary tokens returned from `login` for long-lived tokens and will provide a `UserPrincipal` which can be used to access a users details. If this is not invoked you will not have access to the `UserPrincipal`.
+
+. The callback provided to `login` will be invoked.
+****
+
+[role="secondary"]
+.iOS
+****
+. Use the `login` function to trigger the authentication flow:
```

Comment:
Let's be consistent and use the same wording for each platform, if possible.

Reply:
Yes, as much as possible. But for example in Java you would call it 'Method' while Javascript doesn't really have the concept of a method and you would just say 'function'. Or in Java it would be somewhat verbose to not retrieve the AuthService instance first and just chain the calls while in Swift that's the nice and concise way. So sometimes the procedure will be slightly different.